VKClientRM/VKClientRM/Model/newsfeedGet/Size.swift

```diff
@@ -0,0 +1,16 @@
+// Size.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Размер фотографии
+final class Size: Decodable {
+    /// ссылка по которой хранится фотография
+    let url: String
+
+    // MARK: - CodingKeys
+
+    enum CodingKeys: String, CodingKey {
```

Comment:
```suggestion
    enum CodingKeys: String, CodingKey {
```
А без него будет работать ?

Reply:
нет